maqs-appium/src/test/java/com/cognizantsoftvision/maqs/appium/AppiumUtilitiesUnitTest.java

```diff
@@ -25,7 +24,7 @@
  * Appium Utilities Unit Test class.
  */
 @Ignore
-public class AppiumUtilitiesUnitTest extends BaseGenericTest {
```

Comment:
Why extend base Appium when generic works?

Reply:
reverted back to use baseGenericTest